### ISSUE TITLE
2025-05-13, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#168]
# Today's Arxiv Papers

(1) [Learning Dynamics in Continual Pre-Training for Large Language Models](http://arxiv.org/abs/2505.07796v1), 2025-05-12

(2) [Overflow Prevention Enhances Long-Context Recurrent LLMs](http://arxiv.org/abs/2505.07793v1), 2025-05-12

(3) [Domain Regeneration: How well do LLMs match syntactic properties of text domains?](http://arxiv.org/abs/2505.07784v1), 2025-05-12

(4) [Relative Overfitting and Accept-Reject Framework](http://arxiv.org/abs/2505.07783v1), 2025-05-12

(5) [MLE-Dojo: Interactive Environments for Empowering LLM Agents in Machine Learning Engineering](http://arxiv.org/abs/2505.07782v1), 2025-05-12

(6) [Agent RL Scaling Law: Agent RL with Spontaneous Code Execution for Mathematical Problem Solving](http://arxiv.org/abs/2505.07773v1), 2025-05-12

(7) [Enhancing Code Generation via Bidirectional Comment-Level Mutual Grounding](http://arxiv.org/abs/2505.07768v1), 2025-05-12

(8) [Assessing the Chemical Intelligence of Large Language Models](http://arxiv.org/abs/2505.07735v1), 2025-05-12

(9) [Spoken Language Understanding on Unseen Tasks With In-Context Learning](http://arxiv.org/abs/2505.07731v1), 2025-05-12

(10) [Circuit Partitioning Using Large Language Models for Quantum Compilation and Simulations](http://arxiv.org/abs/2505.07711v1), 2025-05-12

(11) [Codifying Character Logic in Role-Playing](http://arxiv.org/abs/2505.07705v1), 2025-05-12

(12) [PatchTrack: A Comprehensive Analysis of ChatGPT's Influence on Pull Request Outcomes](http://arxiv.org/abs/2505.07700v1), 2025-05-12

(13) [S-GRPO: Early Exit via Reinforcement Learning in Reasoning Models](http://arxiv.org/abs/2505.07686v1), 2025-05-12

(14) [SpecRouter: Adaptive Routing for Multi-Level Speculative Decoding in Large Language Models](http://arxiv.org/abs/2505.07680v1), 2025-05-12

(15) [OnPrem.LLM: A Privacy-Conscious Document Intelligence Toolkit](http://arxiv.org/abs/2505.07672v1), 2025-05-12

(16) [Benchmarking Retrieval-Augmented Generation for Chemistry](http://arxiv.org/abs/2505.07671v1), 2025-05-12

(17) [A Case Study Investigating the Role of Generative AI in Quality Evaluations of Epics in Agile Software Development](http://arxiv.org/abs/2505.07664v1), 2025-05-12

(18) [JobHop: A Large-Scale Dataset of Career Trajectories](http://arxiv.org/abs/2505.07653v1), 2025-05-12

(19) [Neural Brain: A Neuroscience-inspired Framework for Embodied Agents](http://arxiv.org/abs/2505.07634v1), 2025-05-12

(20) [KAQG: A Knowledge-Graph-Enhanced RAG for Difficulty-Controlled Question Generation](http://dx.doi.org/10.36227/techrxiv.174681425.54614303/v1), 2025-05-12

(21) [Concept-Level Explainability for Auditing & Steering LLM Responses](http://arxiv.org/abs/2505.07610v1), 2025-05-12

(22) [TACOS: Temporally-aligned Audio CaptiOnS for Language-Audio Pretraining](http://arxiv.org/abs/2505.07609v1), 2025-05-12

(23) [MiMo: Unlocking the Reasoning Potential of Language Model -- From Pretraining to Posttraining](http://arxiv.org/abs/2505.07608v1), 2025-05-12

(24) [Characterizing the Investigative Methods of Fictional Detectives with Large Language Models](http://arxiv.org/abs/2505.07601v1), 2025-05-12

(25) [Reinforced Internal-External Knowledge Synergistic Reasoning for Efficient Adaptive Search Agent](http://arxiv.org/abs/2505.07596v1), 2025-05-12

(26) [A Multi-Dimensional Constraint Framework for Evaluating and Improving Instruction Following in Large Language Models](http://arxiv.org/abs/2505.07591v1), 2025-05-12

(27) [SecReEvalBench: A Multi-turned Security Resilience Evaluation Benchmark for Large Language Models](http://arxiv.org/abs/2505.07584v1), 2025-05-12

(28) [YuLan-OneSim: Towards the Next Generation of Social Simulator with Large Language Models](http://arxiv.org/abs/2505.07581v1), 2025-05-12

(29) [Direct Density Ratio Optimization: A Statistically Consistent Approach to Aligning Large Language Models](http://arxiv.org/abs/2505.07558v1), 2025-05-12

(30) [Injecting Knowledge Graphs into Large Language Models](http://arxiv.org/abs/2505.07554v1), 2025-05-12

(31) [Towards Requirements Engineering for RAG Systems](http://arxiv.org/abs/2505.07553v1), 2025-05-12

(32) [GRADA: Graph-based Reranker against Adversarial Documents Attack](http://arxiv.org/abs/2505.07546v1), 2025-05-12

(33) [Discrete Visual Tokens of Autoregression, by Diffusion, and for Reasoning](http://arxiv.org/abs/2505.07538v1), 2025-05-12

(34) [RAI: Flexible Agent Framework for Embodied AI](http://arxiv.org/abs/2505.07532v1), 2025-05-12

(35) [QuantX: A Framework for Hardware-Aware Quantization of Generative AI Workloads](http://arxiv.org/abs/2505.07531v1), 2025-05-12

(36) [Byam: Fixing Breaking Dependency Updates with Large Language Models](http://arxiv.org/abs/2505.07522v1), 2025-05-12

(37) [ToolACE-DEV: Self-Improving Tool Learning via Decomposition and EVolution](http://arxiv.org/abs/2505.07512v1), 2025-05-12

(38) [Learning to Reason and Navigate: Parameter Efficient Action Planning with Large Language Models](http://arxiv.org/abs/2505.07500v1), 2025-05-12

(39) [Web-Bench: A LLM Code Benchmark Based on Web Standards and Frameworks](http://arxiv.org/abs/2505.07473v1), 2025-05-12

(40) [A Survey on Collaborative Mechanisms Between Large and Small Language Models](http://arxiv.org/abs/2505.07460v1), 2025-05-12

(41) [Why Uncertainty Estimation Methods Fall Short in RAG: An Axiomatic Analysis](http://arxiv.org/abs/2505.07459v1), 2025-05-12

(42) [Can Generative AI agents behave like humans? Evidence from laboratory market experiments](http://arxiv.org/abs/2505.07457v1), 2025-05-12

(43) [How well do LLMs reason over tabular data, really?](http://arxiv.org/abs/2505.07453v1), 2025-05-12

(44) [LEAD: Iterative Data Selection for Efficient LLM Instruction Tuning](http://arxiv.org/abs/2505.07437v1), 2025-05-12

(45) [A Systematic Literature Review on Neural Code Translation](http://arxiv.org/abs/2505.07425v1), 2025-05-12

(46) [Computational Fact-Checking of Online Discourse: Scoring scientific accuracy in climate change related news articles](http://arxiv.org/abs/2505.07409v1), 2025-05-12

(47) [AI in Money Matters](http://arxiv.org/abs/2505.07393v1), 2025-05-12

(48) [Examining the Role of LLM-Driven Interactions on Attention and Cognitive Engagement in Virtual Classrooms](http://arxiv.org/abs/2505.07377v1), 2025-05-12

(49) [A Preliminary Study of Large Language Models for Multilingual Vulnerability Detection](http://arxiv.org/abs/2505.07376v1), 2025-05-12

(50) [Synthetic Code Surgery: Repairing Bugs and Vulnerabilities with LLMs and Synthetic Data](http://arxiv.org/abs/2505.07372v1), 2025-05-12

(51) [BinMetric: A Comprehensive Binary Analysis Benchmark for Large Language Models](http://arxiv.org/abs/2505.07360v1), 2025-05-12

(52) [QUPID: Quantified Understanding for Enhanced Performance, Insights, and Decisions in Korean Search Engines](http://arxiv.org/abs/2505.07345v1), 2025-05-12

(53) [Private LoRA Fine-tuning of Open-Source LLMs with Homomorphic Encryption](http://arxiv.org/abs/2505.07329v1), 2025-05-12

(54) [Towards Multi-Agent Reasoning Systems for Collaborative Expertise Delegation: An Exploratory Design Study](http://arxiv.org/abs/2505.07313v1), 2025-05-12

(55) [Uncertainty Profiles for LLMs: Uncertainty Source Decomposition and Adaptive Model-Metric Selection](http://arxiv.org/abs/2505.07309v1), 2025-05-12

(56) [L-SWAG: Layer-Sample Wise Activation with Gradients information for Zero-Shot NAS on Vision Transformers](http://arxiv.org/abs/2505.07300v1), 2025-05-12

(57) [AttentionInfluence: Adopting Attention Head Influence for Weak-to-Strong Pretraining Data Selection](http://arxiv.org/abs/2505.07293v1), 2025-05-12

(58) [Semantic Retention and Extreme Compression in LLMs: Can We Have Both?](http://arxiv.org/abs/2505.07289v1), 2025-05-12

(59) [Cache-Efficient Posterior Sampling for Reinforcement Learning with LLM-Derived Priors Across Discrete and Continuous Domains](http://arxiv.org/abs/2505.07274v1), 2025-05-12

(60) [Automated Repair of Ambiguous Natural Language Requirements](http://arxiv.org/abs/2505.07270v1), 2025-05-12

(61) [No Query, No Access](http://arxiv.org/abs/2505.07258v1), 2025-05-12

(62) [SAS-Bench: A Fine-Grained Benchmark for Evaluating Short Answer Scoring with Large Language Models](http://arxiv.org/abs/2505.07247v1), 2025-05-12

(63) [Comet: Accelerating Private Inference for Large Language Model by Predicting Activation Sparsity](http://dx.doi.org/10.1109/SP61157.2025.00182), 2025-05-12

(64) [UAV-CodeAgents: Scalable UAV Mission Planning via Multi-Agent ReAct and Vision-Language Reasoning](http://arxiv.org/abs/2505.07236v1), 2025-05-12

(65) [DynamicRAG: Leveraging Outputs of Large Language Model as Feedback for Dynamic Reranking in Retrieval-Augmented Generation](http://arxiv.org/abs/2505.07233v1), 2025-05-12

(66) [Measuring General Intelligence with Generated Games](http://arxiv.org/abs/2505.07215v1), 2025-05-12

(67) [Benchmarking Ethical and Safety Risks of Healthcare LLMs in China-Toward Systemic Governance under Healthy China 2030](http://arxiv.org/abs/2505.07205v1), 2025-05-12

(68) [PrefillOnly: An Inference Engine for Prefill-only Workloads in Large Language Model Applications](http://arxiv.org/abs/2505.07203v1), 2025-05-12

(69) [Structural Entropy Guided Agent for Detecting and Repairing Knowledge Deficiencies in LLMs](http://arxiv.org/abs/2505.07184v1), 2025-05-12

(70) [Internet of Agents: Fundamentals, Applications, and Challenges](http://arxiv.org/abs/2505.07176v1), 2025-05-12

(71) [One Trigger Token Is Enough: A Defense Strategy for Balancing Safety and Usability in Large Language Models](http://arxiv.org/abs/2505.07167v1), 2025-05-12

(72) [KDH-MLTC: Knowledge Distillation for Healthcare Multi-Label Text Classification](http://arxiv.org/abs/2505.07162v1), 2025-05-12

(73) [HAMLET: Healthcare-focused Adaptive Multilingual Learning Embedding-based Topic Modeling](http://arxiv.org/abs/2505.07157v1), 2025-05-12

(74) [Reassessing Large Language Model Boolean Query Generation for Systematic Reviews](http://arxiv.org/abs/2505.07155v1), 2025-05-12

(75) [Exploring Anthropomorphism in Conversational Agents for Environmental Sustainability](http://arxiv.org/abs/2505.07142v1), 2025-05-11

(76) [KOKKAI DOC: An LLM-driven framework for scaling parliamentary representatives](http://arxiv.org/abs/2505.07118v1), 2025-05-11

(77) [Knowledge Distillation for Enhancing Walmart E-commerce Search Relevance Using Large Language Models](http://dx.doi.org/10.1145/3701716.3715242), 2025-05-11

(78) [RefPentester: A Knowledge-Informed Self-Reflective Penetration Testing Framework Based on Large Language Models](http://arxiv.org/abs/2505.07089v1), 2025-05-11

(79) [Architectural Precedents for General Agents using Large Language Models](http://arxiv.org/abs/2505.07087v1), 2025-05-11

(80) [DriveSOTIF: Advancing Perception SOTIF Through Multimodal Large Language Models](http://arxiv.org/abs/2505.07084v1), 2025-05-11

(81) [Can LLM-based Financial Investing Strategies Outperform the Market in Long Run?](http://arxiv.org/abs/2505.07078v1), 2025-05-11

(82) [ParaView-MCP: An Autonomous Visualization Agent with Direct Tool Use](http://arxiv.org/abs/2505.07064v1), 2025-05-11

(83) [Seed1.5-VL Technical Report](http://arxiv.org/abs/2505.07062v1), 2025-05-11

(84) [DialogueReason: Rule-Based RL Sparks Dialogue Reasoning in LLMs](http://arxiv.org/abs/2505.07049v1), 2025-05-11

(85) [LLM-Augmented Chemical Synthesis and Design Decision Programs](http://arxiv.org/abs/2505.07027v1), 2025-05-11

(86) [MELLM: Exploring LLM-Powered Micro-Expression Understanding Enhanced by Subtle Motion Perception](http://arxiv.org/abs/2505.07007v1), 2025-05-11

(87) [GuidedQuant: Large Language Model Quantization via Exploiting End Loss Guidance](http://arxiv.org/abs/2505.07004v1), 2025-05-11

(88) [Convert Language Model into a Value-based Strategic Planner](http://arxiv.org/abs/2505.06987v1), 2025-05-11

(89) [Web Page Classification using LLMs for Crawling Support](http://arxiv.org/abs/2505.06972v1), 2025-05-11

(90) [From Knowledge to Reasoning: Evaluating LLMs for Ionic Liquids Research in Chemical and Biological Engineering](http://arxiv.org/abs/2505.06964v1), 2025-05-11

(91) [The Wisdom of Agent Crowds: A Human-AI Interaction Innovation Ignition Framework](http://arxiv.org/abs/2505.06947v1), 2025-05-11

(92) [The Distracting Effect: Understanding Irrelevant Passages in RAG](http://arxiv.org/abs/2505.06914v1), 2025-05-11

(93) [Building a Human-Verified Clinical Reasoning Dataset via a Human LLM Hybrid Pipeline for Trustworthy Medical AI](http://arxiv.org/abs/2505.06912v1), 2025-05-11

(94) [Towards Artificial General or Personalized Intelligence? A Survey on Foundation Models for Personalized Federated Intelligence](http://arxiv.org/abs/2505.06907v1), 2025-05-11

(95) [EcoLANG: Efficient and Effective Agent Communication Language Induction for Social Simulation](http://arxiv.org/abs/2505.06904v1), 2025-05-11

(96) [Ecco: Improving Memory Bandwidth and Capacity for LLMs via Entropy-aware Cache Compression](http://arxiv.org/abs/2505.06901v1), 2025-05-11

(97) [Benchmarking and Revisiting Code Generation Assessment: A Mutation-Based Approach](http://arxiv.org/abs/2505.06880v1), 2025-05-11

(98) [Towards Human-Centric Autonomous Driving: A Fast-Slow Architecture Integrating Large Language Model Guidance with Reinforcement Learning](http://arxiv.org/abs/2505.06875v1), 2025-05-11

(99) [Visual Evolutionary Optimization on Combinatorial Problems with Multimodal Large Language Models: A Case Study of Influence Maximization](http://arxiv.org/abs/2505.06850v1), 2025-05-11

(100) [Benign Samples Matter! Fine-tuning On Outlier Benign Samples Severely Breaks Safety](http://arxiv.org/abs/2505.06843v1), 2025-05-11
